build-release.sh

```diff
@@ -104,23 +105,39 @@ publish_nuget_packages() {
   done
 }
 
+can_publish_maven=0
+if [ ! -d "${basedir}/deps/keystore" ]; then
+  echo "Disabling Android library publishing as ${basedir}/deps/keystore doesn't exist."
+else
+  can_publish_maven=1
+fi
+
+publish_maven_library() {
+  if [ $can_publish_maven == 0 ]; then
+    return
+  fi
+  sh build-android/upload-mavencentral.sh
```

Comment:
So the main reason why I kept this as a separate script, is that this script requires `sudo`, but the `build-release.sh` isn't run with `sudo`.

IIRC running it with sudo fails as it's using the wrong user for connecting to HP's Mac over ssh. Can likely be fixed, but needs some experimentation.

Reply:
Ah, right, I remember. So adding `sudo` to this line should be sufficient then? I don't think this would elevate the whole script if we do it this way, would it?